.gitignore

```diff
@@ -37,3 +37,4 @@ MANIFEST
 sosize-*.txt
 pybind11Config*.cmake
 pybind11Targets.cmake
+/*env*
```

Comment:
Is the leading slash intentional?

Reply:
Yes, I'm assuming a virtual environment would be in the main directory or in a build directory